client/my-sites/sharing/connections/account-dialog.jsx

```diff
@@ -187,9 +187,9 @@ class AccountDialog extends Component {
 		// single Keyring connection, and the user must choose which
 		// account to connect.
 		return this.props.translate(
-			'Select the account you wish to authorize. Note that your posts will be shared to the selected account automatically.',
+			"Select the account you'd like to connect. All your new blog posts will be automatically shared to this account. You'll be able to change this option in the editor sidebar when you're writing a post.',
```

Comment:
The linter's complaining about an unterminated string here - this is likely because we've ended the string with a `'` instead of a `"`.

Reply:
Oops, sorry, fixed!

Reply:
Hmm, it looks like the tests are still failing. I don't _think_ it's related to this change though?

Reply:
You can often get a clue about what the problem is if you click on the `Details` link next to the failing group of tests. It shows the linter's complaining about the CSS classes again, this time in the `getConnectedAccountsContent` method. This is the only other place where `className` attributes are set, so if we add some another pair of ESLint comments that should fix them.

Looking at other places where these comments are used, they're commonly placed before and after the `return( ... );` statement, so if you add them above line 147 and below line 166, and move the others to above 206 and below 227, we should be good - fingers crossed!

Reply:
Looks like the linter is happy now. :smile:

Thanks for the help with this!

Reply:
Great stuff! 🎉I'll give this another run-through on my local environment tomorrow, and hopefully I can get this merged for you, thanks!